tests/riscv32/smoke.sh

```diff
@@ -6,28 +6,27 @@
 # See https://llvm.org/LICENSE.txt for license information.
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
-# Test the cross-compiled RISCV targets using Kokoro.
+# Smoke test the cross-compiled RISCV-32 targets.
 
-set -e
-set -x
+set -xeuo pipefail
 
 # Print the UTC time when set -x is on
 export PS4='[$(date -u "+%T %Z")] '
 
 # Run the embedded_library module loader and simple_embedding under QEMU.
 echo "Test elf_module_test_binary"
-pushd "${BUILD_RISCV_DIR?}/runtime/src/iree/hal/local/elf" > /dev/null
-"${QEMU_RV32_BIN?}" -cpu rv32,x-v=true,x-k=true,vlen=256,elen=64,vext_spec=v1.0 \
+pushd "${BUILD_RISCV_DIR}/runtime/src/iree/hal/local/elf" > /dev/null
+"${QEMU_RV32_BIN}" -cpu rv32,x-v=true,x-k=true,vlen=256,elen=64,vext_spec=v1.0 \
```

Comment:
```suggestion
"${QEMU_RV32_BIN}" -cpu rv32 \
```
The IREE programs are not built with vector ops enabled.